certified-connectors/EBMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2582 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "EBMS",
+    "description": "Connect your EBMS data with hundreds of other services!  Contact your Eagle Business Software Account Manager to activate your Power Automate connector.",
+    "version": "1.0",
+    "contact": {
+      "name": "Eagle Business Software",
+      "url": "https://info.eaglebusinesssoftware.com",
+      "email": "info@eaglebusinesssoftware.com"
+    }
+  },
+  "host": "ecc.101010101010101.servicebus.windows.net",
+  "basePath": "/MyEbms/CompanyId/Odata",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/INVENTRY": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```

Comment:
@doniyorniazov empty response schemas are [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses) for operations. 
Actions should have well-defined schemas and responses.
More information and examples can be found in [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).